runtimes/pythonrt/ak_runner/__main__.py

```diff
@@ -5,7 +5,7 @@
 from base64 import b64decode
 from importlib.util import module_from_spec, spec_from_file_location
 from inspect import getsourcelines
-from os import mkdir
+from os import chdir, mkdir
```

Comment:
IMO `import os` and using `os.xxx()` in the code is much clearer - wdyt?

Reply:
My rule of thumb is that if the function name is self explaining, I'll import the function (e.g. `from urllib.request import urlopen`). Otherwise I'll import in a way that makes it clear (e.g. `json.loads`)

IMO `chdir` and `mkdir` are self explaining.